### PR TITLE
button label override

### PR DESCRIPTION
**Pull request by pculka**
add the possibility to override button labels in the same logic as it is with button classes

### CODE REVIEW

File: Resources/views/FormFlow/buttons.html.twig [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
+    {#
+        Default button (the one trigged by pressing the enter/return key) must be defined first.
+        Thus, all buttons are defined in reverse order and will be reversed again via CSS.
+        See http://stackoverflow.com/questions/1963245/multiple-submit-buttons-specifying-default-button
+    #}
+    {%- set craue_label = {
+            button_finish: craue_formflow_button_label_finish | default('button.finish'),
+            button_next: craue_formflow_button_label_next | default('button.next'),
+            button_back: craue_formflow_button_label_back | default('button.back'),
+            button_reset: craue_formflow_button_label_back | default('button.reset')
```
Comment (craue) [inline Resources/views/FormFlow/buttons.html.twig]: I guess this is meant to be `craue_formflow_button_label_reset`.

**Comment by craue** [comment]
Could you avoid changing the indentation? I'm using tabs instead of spaces.
And could you rebase to current master? Quite a lot happened recently, so tests should pass now.

**Comment by pculka** [comment]
Updated with upstream/master and fixed the indentation/codestyle

File: Resources/views/FormFlow/buttons.html.twig [hunk 2]
```diff
@@ -5,7 +5,14 @@
 		Thus, all buttons are defined in reverse order and will be reversed again via CSS.
 		See http://stackoverflow.com/questions/1963245/multiple-submit-buttons-specifying-default-button
 	#}
-
+	{%-
+		set craue_label = {
```
Comment (craue) [inline Resources/views/FormFlow/buttons.html.twig]: Could you put these two lines...

File: Resources/views/FormFlow/buttons.html.twig [hunk 3]
```diff
@@ -5,7 +5,14 @@
...
 	#}
-
+	{%-
+		set craue_label = {
+			button_finish: craue_formflow_button_label_finish | default('button.finish'),
+			button_next: craue_formflow_button_label_next | default('button.next'),
+			button_back: craue_formflow_button_label_back | default('button.back'),
+			button_reset: craue_formflow_button_label_reset | default('button.reset')
+		}
+	-%}
```
Comment (craue) [inline Resources/views/FormFlow/buttons.html.twig]: ...and these two lines onto one each? Similar to the other multiline `set` tag below.

**Comment by craue** [comment]
Could you also add documentation and tests for this, similar to #126?

File: Resources/views/FormFlow/buttons.html.twig (hunk 3 above)
Comment (craue) [inline Resources/views/FormFlow/buttons.html.twig]: But why do you define these new variables at all? Using them inline would be fine for me as well.

**Comment by craue** [comment]
@pculka, do you still want to work on this?

**Comment by pculka** [comment]
Hi, yes, but I'm in holidays atm, will resume work once back. Want to finish this

**Comment by craue** [comment]
@pculka, I absolutely don't begrudge you a long vacation, but are there any news? :smirk:

**Comment by pculka** [comment]
ooon it, sorry, was quite busy in work lately

**Comment by pculka** [comment]
hmmm no idea how to integrate the tests properly, any idea how to see if the tests did cover the code? No idea how to test .twig

**Comment by craue** [comment]
@pculka, see https://github.com/craue/CraueFormFlowBundle/pull/200#issuecomment-165298921 :smirk:

**Pull request closed without merging**